numpy/core/src/multiarray/multiarraymodule.c

```diff
@@ -1666,13 +1666,14 @@ _array_fromobject(PyObject *NPY_UNUSED(ignored), PyObject *args, PyObject *kws)
     }
 
 full_path:
-    if (!PyArg_ParseTupleAndKeywords(args, kws, "O|O&O&O&O&i:array", kwd,
+    if (!PyArg_ParseTupleAndKeywords(args, kws, "O|O&O&O&O&ii:array", kwd,
```

Comment:
```suggestion
    if (!PyArg_ParseTupleAndKeywords(args, kws, "O|O&O&O&O&$ii:array", kwd,
```
Let's make these keyword-only

Reply:
Done